crates/bevy_audio/src/audio_output.rs

```diff
@@ -51,11 +53,40 @@ pub struct PlaybackDespawnMarker;
 #[derive(Component)]
 pub struct PlaybackRemoveMarker;
 
+#[derive(SystemParam)]
+pub(crate) struct SpatialListenerSystemParam<'w, 's> {
+    pub(crate) query: Query<'w, 's, (&'static GlobalTransform, &'static SpatialListener)>,
+    pub(crate) spatial_scale: Res<'w, SpatialScale>,
+}
+impl<'w, 's> SpatialListenerSystemParam<'w, 's> {
+    pub(crate) fn get_ear_positions(&self) -> (Vec3, Vec3) {
+        let mut listener_iter = self.query.iter();
+        let (left_ear, right_ear) = listener_iter
+            .next()
```

Comment:
this should use `get_single`

Reply:
warning is already handled somewhere else 👍

Reply:
For anyone else reading, there was some discussion about this over on discord: https://discord.com/channels/691052431525675048/1158058216882712677

I added some comment. Let me know what you think.